src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -641,11 +638,10 @@ export class ProgrammingExerciseUpdateComponent implements AfterViewInit, OnDest
             });
         }
         if (this.isImportFromFile) {
+            // TODO this might need to be adjusted
```

Comment:
TODO?

Reply:
This is outdated, the import from files does not work properly currently, before adding the support for that the import should be fixed (I had a short look at the issue and it seems out of scope to me, considering that the PR is already quite big) [Programming exercises: Build results not available after file import #8562](https://github.com/ls1intum/Artemis/issues/8562)